pkgs/os-specific/linux/audit/default.nix

```diff
@@ -68,12 +62,12 @@ stdenv.mkDerivation (finalAttrs: {
 
   enableParallelBuilding = true;
 
-  meta = {
+  meta = with lib; {
     homepage = "https://people.redhat.com/sgrubb/audit/";
     description = "Audit Library";
     changelog = "https://github.com/linux-audit/audit-userspace/releases/tag/v${finalAttrs.version}";
-    license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ AndersonTorres ];
-    platforms = lib.platforms.linux;
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [ AndersonTorres ];
+    platforms = platforms.linux;
```

Comment:
Minor, but this should either be forked out into a separate commit, or not done at all.

Reply:
It does not break the semantics, then there is no difference in doing this in a separate commit.

But let's not pursue this bikeshedding.

Reply:
This is my personal preference, based on the fact I am not fond of `with` - as explained many times, it is an anti-pattern:

https://nix.dev/recipes/best-practices#with-scopes

Reply:
Yeah, its very much an anti-pattern. But inside of places like `meta` its one of the few places i find it okay.

Ultimately, this refactor should either be done in another commit, or not at all, since the `meta` does not do any damage being one way or the other.